src/util/notifications_util.js

```diff
@@ -35,13 +35,14 @@ class Notificator {
      * @param {Object} options
      */
 
-    constructor({name, fs_context, connect_files_dir, nc_config_fs}) {
+    constructor({name, fs_context, connect_files_dir, nc_config_fs, batch_size}) {
         this.name = name;
         this.connect_str_to_connection = new Map();
         this.notif_to_connect = new Map();
         this.fs_context = fs_context ?? get_process_fs_context();
         this.connect_files_dir = connect_files_dir ?? DEFAULT_CONNECT_FILES_DIR;
         this.nc_config_fs = nc_config_fs;
+        this.batch_size = batch_size || config.NOTIFICATION_BATCH || 10;
```

Comment:
I thought you will obviously remove it so I did not give comment on this.
Here now you can remove 10. right?

Reply:
Might be over-protective in practice.
In theory someone might nullify config.NOTIFICATION_BATCH.